shell/platform/windows/testing/mock_window.h

```diff
@@ -6,15 +6,15 @@
 #define FLUTTER_SHELL_PLATFORM_WINDOWS_TESTING_MOCK_WIN32_WINDOW_H_
 
 #include "flutter/fml/macros.h"
+#include "flutter/shell/platform/windows/flutter_window.h"
 #include "flutter/shell/platform/windows/testing/test_keyboard.h"
-#include "flutter/shell/platform/windows/window.h"
 #include "gmock/gmock.h"
 
 namespace flutter {
 namespace testing {
 
 /// Mock for the |Window| base class.
-class MockWindow : public Window {
+class MockWindow : public FlutterWindow {
```

Comment:
Consider renaming this type to `MockFlutterWindow` and this file to `mock_flutter_window`. I don't feel strongly about this though.